torch/__init__.py

```diff
@@ -278,6 +278,12 @@ def is_tensor(obj):
 
     Args:
         obj (Object): Object to test
+    Example::
+
+        >>>import torch
+        >>>x=torch.tensor([1, 2, 3])
+        >>>torch.is_tensor(x)
+        >>>True
```

Comment:
this doesn't look correct, the True shouldn't have `>>>`, also don't import torch in the example.

Reply:
Thanks @gchanan,Submitted New patch as per your suggestion,Some jobs are failing,We dont have access to trigger them again,can you please suggest how to proceed next